api/test/integration/test_agent_GET_endpoints.tavern.yaml

```diff
@@ -4555,7 +4592,7 @@ stages:
             - count: 2
               node_name: 'unknown'
           failed_items: []
-          total_affected_items: 2
+          total_affected_items: 1
```

Comment:
This was a bug actually, we returned one element but a `total_affected_items` of `2`